x-pack/legacy/plugins/siem/public/apps/plugin.tsx

```diff
@@ -32,6 +33,10 @@ export class Plugin {
   }
 
   public start(core: StartCore, plugins: StartPlugins) {
+    // TODO(rylnd): These are unknown by uiSettings by default
+    core.uiSettings.set(DEFAULT_KBN_VERSION, '8.0.0');
```

Comment:
is it better to be hardcoded or it should come back from chrome until NP has them

Reply:
Agreed, this definitely should not be hardcoded in the long term. I have a TODO to resolve this before 7.6 releases. [Conversation](https://github.com/elastic/kibana/pull/52783#discussion_r358563805) is ongoing.